Doc/reference/lexical_analysis.rst

```diff
@@ -482,7 +482,9 @@ In plain English: Both types of literals can be enclosed in matching single quot
 of three single or double quotes (these are generally referred to as
 *triple-quoted strings*).  The backslash (``\``) character is used to escape
 characters that otherwise have a special meaning, such as newline, backslash
-itself, or the quote character.
+itself, or the quote character. It can also be used to give special meaning to
+otherwise ordinary characters like ``n``, which means 'newline' when escaped
+(``\n``). See :ref:`escape sequences <escape-sequences>` below for examples.
```

Comment:
GitHub isn't letting me do a suggestion here because the range includes a deleted line, but I think the meaning here would be clearer if we swapped the order of the two lines:

```
*triple-quoted strings*).  The backslash (``\``) character can be used to give
special meaning to otherwise ordinary characters like ``n``, which means
'newline' when escaped (``\n``). It can also be used to escape
characters that otherwise have a special meaning, such as newline, backslash
itself, or the quote character.
See :ref:`escape sequences <escape-sequences>` below for examples.
```

Reply:
Makes sense, going from ordinary to more special cases